doc/devel/communication_guide.rst

```diff
@@ -4,11 +4,32 @@
 Communication guidelines
 ========================
 
+The Scientific Python community uses various communications platforms to stay
+updated on new features and projects, to contribute by telling us what is on
+their mind and suggest issues and bugs, and to showcase their use cases and the
+tools they have built.
+
+Our communications strategy is foremost guided by our :ref:`mission-statement`:
+
+* We demonstrate that we care about visualization as a practice
+* We deepen our practice and the community’s capacity to support users,
+  facilitate exploration, produce high quality viz, and be understandable and
+  extensible
+* We showcase advanced use of the library without adding maintenance burden to
+  the documentation and show recognition for work that may not be in the
+  “what’s new” section.
+* We use communications platforms to maintain relationships with contributors
+  who may no longer be active on GitHub, build relationships with potential
+  contributors, and connect with other projects and communities who use
+  Matplotlib.
+* In prioritizing understandability and extensiblity, we empower people working
+  with Matplotlib to feel like they are a part of our community and that they are
+  doing valuable community building labor.
```

Comment:
Is "working with Matplotlib" already valueable community building? Maybe "building on top of Matplotlib"? Also, you can leave out "labor", "doing valuable community building." is enough.